pkg/cmd/browse/browse.go

```diff
@@ -59,8 +59,16 @@ func NewCmdBrowse(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Co
 	}
 
 	cmd := &cobra.Command{
-		Long:  "Open the GitHub repository in the web browser.",
-		Short: "Open the repository in the browser",
+		Short: "Open repositories, issues, pull requests, and more in the browser",
+		Long:  heredoc.Doc(`
+			Transition from the terminal to the web browser to view and interact with:
+
+			- issues
+			- pull requests
+			- repository content
+			- repository profile
+			- repository settings
```

Comment:
I think we should sentence case these 🙂 

```suggestion
			Transition from the terminal to the web browser to view and interact with:

			- Issues
			- Pull requests
			- Repository content
			- Repository profile
			- Repository settings
```